site/content/kapp-controller/docs/latest/private-registry-auth.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Authenticating to Private Registires
+---
+
+In order for kapp-controller to pull imgpkg bundles or images from registries requiring authentication, 
+kapp-controller supports a workflow to more easily manage Kubernetes secrets and use them as part of working 
+with PackageRepositories or PackageInstalls. 
+
+In addition to having kapp-controller installed, when [secretgen-controller](https://github.com/vmware-tanzu/carvel-secretgen-controller) 
+is installed on the same cluster, kapp-controller will create secrets that are used when PackageRepositories or 
```

Comment:
appeals to me to add the word 'placeholder', as in:  "...kapp-controller will create placeholder secrets that are used..."

Reply:
Thanks, added and also linked to placeholder secret docs.